command/certificate/needsRenewal.go

```diff
@@ -7,59 +7,55 @@ import (
 	"github.com/smallstep/cli/errs"
 	"github.com/urfave/cli"
 	"io/ioutil"
-	"os"
 	"strconv"
 	"strings"
 	"time"
 )
 
+const defaultPercentUsedThreshold = 66
+
 func needsRenewalCommand() cli.Command {
 	return cli.Command{
 		Name:      "needs-renewal",
 		Action:    cli.ActionFunc(needsRenewalAction),
 		Usage:     `Check if a certificate needs to be renewed`,
-		UsageText: `**step certificate needs-renewal** <crt_file or host_name> [**--expires-in <duration>]`,
-		Description: `**step certificate needs-renewal** Checks certificate expiration from file or from a host if 
-		the certificate is over 66% of its lifetime. If the certificate needs renewal this command will return '0'.
-		If the certificate is not far enough into its life, it will return '1'. If validation fails, or if an error occurs, 
-		this command will produce a non-zero return value.
+		UsageText: `**step certificate needs-renewal** <crt_file or host_name> [**--expires-in**=<duration>]`,
```

Comment:
Use `cert_file` here and in other places where you refer to this input. Over time we came to the custom of using `cert` if it's in documentation that the user sees because that's the standard other popular tools use (e.g. openssl, curl), and `crt` if we're just naming variables internally in our code.